package.json

```diff
@@ -47,6 +47,8 @@
         "tsc": "tsc"
     },
     "dependencies": {
+        "@testing-library/react": "14.0.0",
+        "@testing-library/user-event": "14.4.3",
```

Comment:
So when a package.json says a package is a dependency, this means it is a runtime dependency, and should be shipped with the actual library code. 

In this case, these are not runtime dependencies, as they're only used for testing. 

You probably want to selectively disable the lint rule for no-extraneous-depenencies instead.